DataFormats/Scouting/interface/Run3ScoutingMuon.h

```diff
@@ -0,0 +1,305 @@
+  float trk_dzError() const { return trk_dzError_; }
+  float trk_qoverpError() const { return trk_qoverpError_; }
+  float trk_lambdaError() const { return trk_lambdaError_; }
+  float trk_phiError() const { return trk_phiError_; }
+  float trk_dsz() const { return trk_dsz_; }
+  float trk_dszError() const { return trk_dszError_; }
+  //add off-diagonal covariance matrix parameter, the above "Error" variables correspond to the diagonal, enum for Cov matrix (qoverp, lambda, phi, dxy, dsz), see https://github.com/cms-sw/cmssw/blob/CMSSW_11_2_X/DataFormats/TrackReco/src/TrackBase.cc for details
+  float trk_qoverp_lambda_cov() const { return trk_qoverp_lambda_cov_; }
+  float trk_qoverp_phi_cov() const { return trk_qoverp_phi_cov_; }
+  float trk_qoverp_dxy_cov() const { return trk_qoverp_dxy_cov_; }
+  float trk_qoverp_dsz_cov() const { return trk_qoverp_dsz_cov_; }
+  float trk_lambda_phi_cov() const { return trk_lambda_phi_cov_; }
+  float trk_lambda_dxy_cov() const { return trk_lambda_dxy_cov_; }
+  float trk_lambda_dsz_cov() const { return trk_lambda_dsz_cov_; }
+  float trk_phi_dxy_cov() const { return trk_phi_dxy_cov_; }
+  float trk_phi_dsz_cov() const { return trk_phi_dsz_cov_; }
+  float trk_dxy_dsz_cov() const { return trk_dxy_dsz_cov_; }
+  float trk_vx() const { return trk_vx_; }
+  float trk_vy() const { return trk_vy_; }
+  float trk_vz() const { return trk_vz_; }
+  reco::HitPattern trk_hitPattern() const { return trk_hitPattern_; }
+  std::vector<int> vtxIndx() const { return vtxIndx_; }
+
+private:
+  float pt_;
+  float eta_;
+  float phi_;
+  float m_;
+  unsigned int type_;
+  int charge_;
+  float normalizedChi2_;
+  float ecalIso_;
+  float hcalIso_;
+  float trackIso_;
+  int nValidStandAloneMuonHits_;
+  int nStandAloneMuonMatchedStations_;
+  int nValidRecoMuonHits_;
+  int nRecoMuonChambers_;
+  int nRecoMuonChambersCSCorDT_;
+  int nRecoMuonMatches_;
+  int nRecoMuonMatchedStations_;
+  unsigned int nRecoMuonExpectedMatchedStations_;
+  unsigned int recoMuonStationMask_;
+  int nRecoMuonMatchedRPCLayers_;
+  unsigned int recoMuonRPClayerMask_;
+  int nValidPixelHits_;
+  int nValidStripHits_;
+  int nPixelLayersWithMeasurement_;
+  int nTrackerLayersWithMeasurement_;
+  float trk_chi2_;
+  float trk_ndof_;
+  float trk_dxy_;
+  float trk_dz_;
+  float trk_qoverp_;
+  float trk_lambda_;
+  float trk_pt_;
+  float trk_phi_;
+  float trk_eta_;
+  float trk_dxyError_;
+  float trk_dzError_;
+  float trk_qoverpError_;
+  float trk_lambdaError_;
+  float trk_phiError_;
+  float trk_dsz_;
+  float trk_dszError_;
+  float trk_qoverp_lambda_cov_;
+  float trk_qoverp_phi_cov_;
+  float trk_qoverp_dxy_cov_;
+  float trk_qoverp_dsz_cov_;
+  float trk_lambda_phi_cov_;
+  float trk_lambda_dxy_cov_;
+  float trk_lambda_dsz_cov_;
+  float trk_phi_dxy_cov_;
+  float trk_phi_dsz_cov_;
+  float trk_dxy_dsz_cov_;
+  float trk_vx_;
+  float trk_vy_;
+  float trk_vz_;
+  reco::HitPattern trk_hitPattern_;
```

Comment:
hi - sorry just noticing - this would be a first instance of adding complex classes to the scouting raw data formats. We had previously agreed this was hard to promise support of long term (that obviously can be reconsidered)